custom/conf/app.example.ini

```diff
@@ -1262,6 +1262,12 @@ ROUTER = console
 ;; List of file extensions that should be rendered/edited as Markdown
 ;; Separate the extensions with a comma. To render files without any extension as markdown, just put a comma
 ;FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd
+;;
+;; Enables math inline and block detection
+;ENABLE_MATH = true
+;;
+;; Enables in addition inline block detection using single dollars
+;ENABLE_INLINE_DOLLAR_MATH = false
```

Comment:
Maybe `ENABLE_INLINE_MATH` would be a better name. Would certainly work better if there is non-dollar stuff.

Reply:
There is non-dollar math.

```
\( x = 5\)
```

is inline LaTeX style.

```
$ x = 5 $
```

is inline dollar or TeX style.

INLINE_DOLLAR_MATH is the correct name. INLINE_TEX_MATH might be more technically correct but is confusing to people who don't think of LaTeX and TeX as being separate.

Reply:
I see, the second example works even with inline dollar disabled.